Cargo.lock

```diff
@@ -969,28 +926,27 @@ version = "0.1.2"
 dependencies = [
  "app_dirs",
  "ethereum-types",
- "home",
+ "home 0.3.4",
  "journaldb",
 ]
 
 [[package]]
 name = "docopt"
-version = "1.0.2"
+version = "1.1.0"
```

Comment:
FYI, building this panic's for me on nightly , https://github.com/rust-lang/rust/issues/70302

Reply:
Fixed by https://github.com/rust-lang/rust/pull/70051